lifecycle_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright The OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !coraza.rule.multiphase_evaluation
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tetratelabs/proxy-wasm-go-sdk/proxywasm/proxytest"
+	"github.com/tetratelabs/proxy-wasm-go-sdk/proxywasm/types"
+
+	"github.com/corazawaf/coraza-proxy-wasm/wasmplugin"
+)
+
+func TestBodyRulesWithoutBody(t *testing.T) {
```

Comment:
I didn't do a detailed diff between the two tests but is it possible to use one file / test code and tweak the assertions for multiphase vs not? I think that'll make it much clearer "the difference" between the two modes. If the tests are drastically different, maybe that's not possible though.